packages/manager/modules/manager-pci-common/package.json

```diff
@@ -1,70 +1,90 @@
 {
   "name": "@ovh-ux/manager-pci-common",
   "version": "0.9.1",
-  "private": true,
-  "sideEffects": true,
+  "private": false,
+  "description": "Public Cloud Common components",
+  "homepage": "https://github.com/ovh/manager/blob/master/packages/manager/modules/manager-pci-common/README.md",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/manager-pci-common"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
   "type": "module",
-  "main": "./src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
-    "dev": "tsc",
+    "build": "vite build",
+    "clean": "rimraf node_modules && rimraf dist && yarn cache clean",
```

Comment:
**question:** It is anyways removed with the global `yarn clean` command. Is this command to handle the clean only for the lib?

Reply:
@anooparveti 
Yes, this script only concerns the lib.
Before building, if we want to clean the package's node_modules and the dist folder, it's better to not remove the whole monorepo elements and this ensures that lib has been correctly cleaned.